besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -557,6 +557,22 @@ void setBannedNodeIds(final List<String> values) {
       description = "Set if node will perform mining (default: ${DEFAULT-VALUE})")
   private final Boolean isMiningEnabled = false;
 
+  @Option(
```

Comment:
This is the last open review comment - please let me know how you would like to proceed.

Reply:
Having all the related options starting with `--miner-stratum-*` is good (i.e. `miner-stratum-enabled`, `miner-stratum-host`, `miner-stratum-port`).

This is ideal since `--miner-coinbase` and `--miner-extra-data` are still required. Seems like all of these options are a sub-set of `miner`.  

The model is that `--miner-enabled` turns mining on/off. `--miner-stratum-enabled` changes the type of mining. So, `--miner-stratum-enabled` is dependent on `--miner-enabled`.

Reply:
So we will need to add the logic so that 
`--mining-enabled` == cool, do CPU mining
`--mining-enabled --mining-stratum-enabled` == cool, do Stratum mining and not CPU mining
just `--mining-stratum-enabled` == not cool, complain about needing the `--mining-enabled` flag.  

My quick looks shows this logic is in place, it just needs updated text.

Reply:
OK, changes have been applied towards that. Note it's `--miner`, not `--mining`.
I changed the logic to match the behavior intended.